lib/engines/content_block_manager/app/components/content_block_manager/content_block/document/show/document_timeline_component.rb

```diff
@@ -28,16 +28,20 @@ def show_to_user?(version)
   def title(version)
     case version.state
     when "published"
-      version.state.capitalize
+      if version == first_published_version
+        "#{version.item.block_type.humanize} created"
+      else
+        version.state.capitalize
+      end
     when "scheduled"
       "Scheduled for publishing on #{version.item.scheduled_publication.to_fs(:long_ordinal_with_at)}"
     else
       "#{version.item.block_type.humanize} #{version.state}"
     end
   end
 
-  def first_created_edition
-    content_block_versions.last
+  def first_published_version
+    @first_published_version ||= content_block_versions[-2]
```

Comment:
This seems a bit magic, could we do something like:

```
@first_published_version ||= content_block_versions.order(&:created_at).find { |v| v.event == "published" }
```

Reply:
that's a good call actually, it's more specific instead of relying on the order of versions always being the same

Reply:
hmm ok I'm having trouble with the rails way of sorting... it's an array so I need to `sort_by` and I want to do by descending because I want the second to last oldest, but it doesn't like this? 
`@first_published_version ||= content_block_versions.sort_by{|v| -v[:created_at]}.drop(1).find { |v| v.state == "published" }`

Reply:
Try this maybe?

```
@first_published_version ||= content_block_versions.sort_by{|v| v[:created_at]}.reverse.drop(1).find { |v| v.state == "published" }
```

Reply:
Do you need the `drop(1)` as you're filtering for `published` events?

Reply:
On reflection, it might be more performant to filter first:

```
@first_published_version ||= content_block_versions.filter { |v| v.state == "published" }.sort_by{ |v| v[:created_at]}.reverse.first
```

Reply:
> Do you need the drop(1) as you're filtering for published events?
oh yeahh, d'oh!

Reply:
Awesome! 🙌